### PR TITLE
EventHub: Added swtich parameter - EnableKafka in New-AzureRmEventHubNamespace

### PR DESCRIPTION
**Pull request by rrama-Microsoft**
## Description

Added swtich parameter - EnableKafka in New-AzureRmEventHubNamespace to support Kafka enable namepaces

## Checklist

- [X] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [X] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by maddieclayton** [comment]
@azuresdkci Add to whitelist

**Comment by rrama-Microsoft** [comment]
@maddieclayton Pulled the latest, the .netcore.cspoj issue is resolved. now the EventGrid test are failing due to older version of EventHub Management client. I guess the same will be with IotHub. can you please take look.

File: src/ResourceManager/EventHub/Commands.EventHub/help/New-AzureRmEventHubNamespace.md [hunk 1]
```diff
@@ -216,20 +240,23 @@ Accept wildcard characters: False
...
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
+For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
 
 ## INPUTS
 
 ### System.String
 
-### System.Nullable`1[[System.Int32, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]]
+System.Nullable`1[[System.Int32, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]]
```
Comment (markcowl) [inline src/ResourceManager/EventHub/Commands.EventHub/help/New-AzureRmEventHubNamespace.md]: Please remove everything under the INPUTS heading - none of these are actually input types

Comment (rrama-Microsoft) [reply src/ResourceManager/EventHub/Commands.EventHub/help/New-AzureRmEventHubNamespace.md]: resolved. removed everything under INPUTS

**Pull request merged**